lib/ansible/modules/cloud/vmware/vmware_vswitch.py

```diff
@@ -32,11 +32,13 @@
     switch_name:
         description:
             - vSwitch name to add
+            - Alias C('switch') is added in version 2.4
```

Comment:
Actually nics and switch are not aliases, they are the default name (which I think is fine). But the documentation (and this message) does not reflect that.